.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     'prettier/prettier': ['error'],
     '@typescript-eslint/indent': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
I don't quite understand your motivations for turning this off - could we leave this as is unless it's really causing problems?

Reply:
Of course! I turned this off because my text editor was showing inline warnings for all the `any`s. For example eslint was telling me these were a problem:
https://github.com/jackfranklin/test-data-bot/blob/2f52d66470f58e02051fed3f9d851ba457c57cc1/src/index.ts#L19-L20

It's not really a big problem though. I'll revert this change!